azure-pipelines.yml

```
@@ -0,0 +1,68 @@
+name: $(BuildDefinitionName)_$(Date:yyyyMMdd)$(Rev:.rr)
+variables:
+  CI_NAME: Azure Pipelines
+  CI_BUILD_ID: $(Build.BuildId)
+  CI_BUILD_URL: "https://dev.azure.com/sunpy/sunkit-dem/_build/results?buildId=$(Build.BuildId)"
+  CIBW_BUILD: cp36-* cp37-* cp38-*
```

```suggestion
  CIBW_BUILD: cp37-* cp38-*
```
If we're using Python 3.7 as the minimum version, then I _think_ we can remove the `cp36-*` which (maybe?) refers to the CPython 3.6.  I'm not completely sure with what the `cp36-*` is doing here though.

---

yep, this is building wheels for CPython 3.6